### ISSUE TITLE
Trouble using cellEditor InputParameterizedCellEditor in Ag-Grid v25

### ISSUE THREAD

**Issue by flunosalvador** [avallete/ag-grid-typed-editor#3]
Hi,

I am currently trying to use the cellEditor `InputParameterizedCellEditor` after upgrading Ag-Grid from v22 to v25 (all of this under the Angular framework). There were not any problems using the former, but there seems to be some kind of trouble with the later version . 

Apparently, everything goes right when compiling the application and when loading grids for editing, but for some reason, when I try to edit a cell using the cell editor aforementioned, the cell is unresponsive (it is not entering into edit mode after pressing enter key or double-clicking it, it remains like if it was noneditable when that is not the case at all). 

As I said, previously there not any issues regarding this when using Ag-Grid v22, so my guess is that this library probably it is not up to date with the incoming changes from the newest versions.

Are there any plans to maintain this library or is there any suggestion regarding the preceding problem I have described?

Thanks in advance.

**Comment by jgragim** [comment]
Yeah, for me it's the same, after upgrading to ag-grid v25 this plugin is not working anymore.
